code-challenges.js

```diff
@@ -4,61 +4,103 @@
 // Use the test variables provided.
 
 var num1 = 15
+
 // Expected output: "15 is divisible by three"
 
 var num2 = 0
 // Expected output: "0 is divisible by three"
 
 var num3 = -7
-// Expected output: "-7 is not divisble by three"
 
+// Expected output: "-7 is not divisble by three"
+// const myDiv = (num) => {
+//     if (num % 3 ==0){
+//     return `${num} is divisble by three`
+//  } else{ return `${num} is not divisble by three`
 
+//     }
+// }
+// console.log(myDiv(num1))
+// console.log(myDiv(num2))
+// console.log(myDiv(num3))
 
 
 
 // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
-var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
-
-
+// var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
+// // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
+// const upperCase = (array) => {
+//     return array.map(value => value.charAt(0).toUpperCase() + value.substr(1))
+// }
+ 
+// console.log(upperCase(randomNouns))
 
 
 // --------------------3) Create a function that takes in the mixedDataArray array and returns an array with ONLY NUMBERS sorted from least to greatest.
 
 var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
 // Expected output: [-8, 0, 8, 46, 59, 90, 107]
 
-
+// const onlyNum = (array) => {
+//     let newArry = []
+//         newArry = array.sort((a,b) => a- b).filter(value => typeof value ==="number")
+//         return newArry
+// }
+// console.log(onlyNum(mixedDataArray))
```

Comment:
you can save yourself some lines of code by returning line 48 outright, instead of creating a new empty string, assigning it some values, then returning it later. 
ex:  `return array.sort((a,b) => a- b).filter(value => typeof value ==="number")`
your code still totally works, just a suggestion!